automation/test.sh

```diff
@@ -70,11 +70,12 @@ function latest_version() {
 export KUBEVIRT_VERSION=$(latest_version "kubevirt")
 
 # Latest released CDI version
-export CDI_VERSION=$(latest_version "containerized-data-importer")
+#export CDI_VERSION=$(latest_version "containerized-data-importer")
```

Comment:
If it's no longer required, we should drop this line to keep the file clean.

Reply:
This is just testing PR, which should investigate where is the problem in CI. This change will not be part of result PR